src/bootstrap/bootstrap.py

```diff
@@ -1041,7 +1041,7 @@ def check_vendored_status(self):
                             "--sync ./compiler/rustc_codegen_gcc/Cargo.toml " \
                             "--sync ./src/bootstrap/Cargo.toml "
```

Comment:
If we're going to have an x.py vendor, it seems like we might as well move this list into the source? Is there a reason to run it and not use all of these directories?

Reply:
Makes sense.